ui/src/message_popup/author/edit_question_page.jsx

```diff
@@ -11,15 +11,67 @@ export default React.createClass({
   displayName: 'EditQuestionPage',
 
   propTypes: {
-    questionId: React.PropTypes.string.isRequired
+    questionId: React.PropTypes.string.isRequired,
+    loaded: React.PropTypes.bool,
+    allQuestions: React.PropTypes.object,
+    onReloadQuestions: React.PropTypes.func
+  },
+
+  componentDidMount(){
+    this.props.onReloadQuestions();
```

Comment:
Does making the initial request make more sense in the loader (so it would always request on mounting)?